app/models/article.rb

```diff
@@ -0,0 +1,4 @@
+class Article < ApplicationRecord
+  enum category: [:experience, :story]
```

Comment:
So I don't understand now. If this is the category of an article, what will you do when you get to the feature `API provides functionality to view articles by category`? Will you sort them by these two then? In that case this is out of scope for this feature and should be presented when you get there. 
Otherwise, if this is not that category you are referring to, then you need to change the name here. It is important to keep a name standard so that not only you but also an external person understands what the attributes are, so for example have it as `article_type` or something similar.

Reply:
That's where the potential `sub_category` would come in. `Article_type` also seems fitting :)